Sharpmake/Project.cs

```diff
@@ -173,6 +173,11 @@ public int SourceFilesFiltersCount
 
         public Strings XResourcesResw = new Strings();
 
+        /// <summary>
+        /// Enable or disable the property [EnableDefaultItems] in NetCore Project Schema
+        /// </summary>
+        public bool EnableDefaultItems { get; set; } = false;
```

Comment:
Do you know if this property only applies to c#? In that case, I would move it to the c# specific class `public class CSharpProject`

Reply:
I am not sure, but it seems to be C# only, I will update it.

Reply:
https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#enabledefaultitems

Reply:
done

Reply:
Thanks for the link and making the change! Indeed that's .net SDK only so better in CSharpProject